build/lib/nls.ts

```diff
@@ -112,19 +114,19 @@ globalThis._VSCODE_NLS_MESSAGES=${JSON.stringify(_nls.allNLSMessages)};`),
 			this.emit('end');
 		}));
 
-	return duplex(input, output);
+	return eventStream.duplex(input, output);
 }
 
 function isImportNode(ts: typeof import('typescript'), node: ts.Node): boolean {
 	return node.kind === ts.SyntaxKind.ImportDeclaration || node.kind === ts.SyntaxKind.ImportEqualsDeclaration;
 }
 
-module _nls {
+const _nls = (() => {
```

Comment:
I'm curious why the module has to be converted over.

Reply:
Node only strips out the types when running TypeScript code, so it can't actually understand TS syntax like `namespace`. Syntax that isn't just typings, such as namespace and enums, have to be rewritten for now to use more standard JS